main.go

```diff
@@ -1,38 +1,44 @@
 package main
 
 import (
-	"fmt"
 	"log"
 	"os"
+	"strings"
 
+	"github.com/gin-contrib/cors"
 	"github.com/gin-gonic/gin"
-	"github.com/joho/godotenv"
+	"github.com/yarinBenisty/api-gateway/util"
 	bpb "github.com/yarinBenisty/birthday-service/proto"
 	"google.golang.org/grpc"
 )
 
 const (
-	name           = "name"
-	personalNumber = "personalNumber"
-	date           = "date"
+	// Name parameter
+	Name = "name"
+	// PersonalNumber parameter
+	PersonalNumber = "personalNumber"
+	// Date parameter
+	Date = "date"
 )
 
 // Router that's connecting to the client
 // functions in the bd-service proto file
 type Router struct {
 	client bpb.BirthdayFunctionsClient
+	bpb.UnimplementedBirthdayFunctionsServer
 }
 
 // Func that creates a new client and connects
 // to the bd-service client through the proto file
 func initClientConnection() bpb.BirthdayFunctionsClient {
 
-	envError := godotenv.Load()
-	if envError != nil {
-		log.Fatal("Error loading .env file! error: ", envError)
+	// Loading the dotenv parameters
+	config, err := util.LoadConfig(".")
```

Comment:
Why reload the config all over again?

Reply:
when i tried , i couldn't load it once for the whole file.

Reply:
and i use it inside two different functions